db/data/regions/AU/en.yml

```diff
@@ -8,29 +8,42 @@ en:
             default: Suburb
             optional: Suburb (optional)
           errors:
-            blank: Enter a suburb
-            too_long: The suburb name is too long (maximum is 255 characters)
-            contains_emojis: Suburb cannot contain emojis
-            contains_restricted_characters: Suburb can only contain letters, numbers,
-              local characters, and special characters
-            contains_html_tags: Suburb cannot contain HTML tags.
+            blank_instructional: Enter a suburb
+            blank_informative: Suburb is missing
+            too_long_instructional: Suburb is too long (maximum is 255 characters)
+            too_long_informative: Suburb name is too long
+            contains_emojis_instructional: Suburb can't contain emojis
+            contains_emojis_informative: Suburb can't contain emojis
+            contains_restricted_characters_instructional: Suburb can only contain letters, numbers, local characters, and special
+              characters
+            contains_restricted_characters_informative: Suburb can't contain restricted characters
+            contains_html_tags_instructional: Suburb can't contain HTML
+            contains_html_tags_informative: Suburb can't contain HTML
         province:
           label:
             default: State/territory
             optional: State/territory (optional)
           errors:
-            blank: Select a state/territory
+            blank_instructional: Select a state/territory
+            blank_informative: State/territory is missing
         zip:
           label:
             default: Postcode
             optional: Postcode (optional)
           errors:
-            blank: Enter a postcode
-            invalid_for_country: Enter a valid postcode for %{country}
-            invalid_for_country_and_province: Enter a valid postcode for %{province},
+            blank_instructional: Enter a postcode
+            blank_informative: Postcode is missing
+            invalid_for_country_instructional: Enter a valid postcode for %{country}
+            invalid_for_country_informative: Postcode isn't valid for %{country}
+            invalid_for_country_and_province_instructional: Enter a valid postcode for %{province},
               %{country}
-            too_long: The postcode value is too long
-            contains_emojis: Postcode cannot contain emojis
-            contains_restricted_characters: Postcode can only contain letters, numbers,
+            invalid_for_country_and_province_informative: Postcode doesn't match %{province},
```

Comment:
This seems weird... it should probably be: 

"Postcode isn't valid for %{province}"